metricbeat/module/kubernetes/state_deployment/data.go

```diff
@@ -0,0 +1,64 @@
+package state_deployment
```

Comment:
Why not call the metricset `deployment`if you put it under the `deployment` namespace?